refgenie/refgenie.py

```diff
@@ -508,6 +547,20 @@ def main():
         refgenie_add(rgc, args)
 
     elif args.command == PULL_CMD:
+
+        # check for registry_path format?
+        if args.registry_path:
+            _LOGGER.debug("Found registry_path: {}".format(args.registry_path))
+            parsed_registry_path = parse_identifier_string(args.registry_path)
```

Comment:
is this supposed to be `parse_registry_path`?

Reply:
ok, changed it

Reply:
yes. I changed the name.. thx